src/streams.ts

```diff
@@ -1,4 +1,7 @@
 import { Subject } from 'rxjs'
 import { SimulationTransactionOutput } from './types'
 
-export const simulations$ = new Subject<SimulationTransactionOutput>()
+export const simulations$ = new Subject<{
+  eventId: string
```

Comment:
Will this cause any problems if the `eventId` isn't available since it is optional in the `BaseEventObject`?

Reply:
Good question. The `eventId` sent to the server on all simulation requests and the server will always echo it back. So the `eventId` will be always there for sim events.